beamer/models/claim.py

```diff
@@ -25,25 +25,23 @@ def __init__(
         # sending another transaction
         self.transaction_pending = False
 
+    started = State("Started", initial=True)
     # Claimer is winning
-    claimer_winning = State("ClaimerWinning", initial=True)
+    claimer_winning = State("ClaimerWinning")
     # Challenger is winning
     challenger_winning = State("ChallengerWinning")
-    invalidated = State("Invalidated")
     invalidated_l1_resolved = State("InvalidatedL1Resolved")
     ignored = State("Ignored")
     withdrawn = State("Withdrawn")
 
+    start_challenge = started.to(claimer_winning) | claimer_winning.to(claimer_winning)
```

Comment:
out of curiosity, in which case do we call `start_challenge()` on a claimer_winning state?

Reply:
Ah, when we received a fillHashInvalid event from the fill manager, right?